tool/planet/dns.go

```diff
@@ -74,7 +74,7 @@ func writeEnvDNSAddresses(addr []string, overwrite bool) error {
 
 func updateEnvDNSAddresses(client *kubernetes.Clientset, role agent.Role, serviceCIDR net.IPNet) error {
 	// locate the cluster IP of the kube-dns service
-	masterServices, err := client.CoreV1().Services(metav1.NamespaceSystem).List(metav1.ListOptions{
```

Comment:
Probably should pass the context through from `setupResolver` and use that in both these `updateEnvDNSAddresses` calls.